mrjob/emr.py

```diff
@@ -2680,7 +2681,7 @@ def make_emr_client(self):
 
         # #1799: don't retry faster than EMR checks the API
         return _wrap_aws_client(raw_emr_client,
-                                min_backoff=self._opts['check_cluster_every'])
+                                min_backoff=self._opts['emr_backoff'])
```

Comment:
Whats the difference? Why is emr_backfill better?

Reply:
Also tests?

Reply:
Because `check_cluster_every` is used in several other places and should stay around 30 seconds or so. Example use case:

```
    def _wait_for_step_to_complete
        ...
        while True:
            # don't antagonize EMR's throttling
            log.debug('Waiting %.1f seconds...' %
                      self._opts['check_cluster_every'])
            time.sleep(self._opts['check_cluster_every'])
```

Reply:
I'll add some tests this week.

Reply:
I'm not going to add an specific tests as there are already tests for retrying logic. I will just update the current tests to pass in this parameter.

Reply:
Actually there are already tests! Modifying them now.